doc/sphinx/administration/logs.rst

```diff
@@ -0,0 +1,437 @@
+====
+Logs
+====
+
+.. note::
+   The Bastion comes with a lot of traceability features, you have to ensure that you've done your configuration correctly, so that those logs are kept in a safe place when you need them. It is warmly advised to enable at least the syslog option, and push your logs to a remote syslog server.
```

Comment:
```suggestion
   The Bastion comes with a lot of traceability features, you have to ensure that you've done your configuration correctly so that those logs are kept in a safe place when you need them. It is warmly advised to enable at least the syslog option, and push your logs to a remote syslog server.
```